tests/test-program.py

```diff
@@ -181,6 +181,12 @@ def transform_messages(self, testcase, context=None):
 
         out = open(os.path.join(self.test_folder, f'{testcase}.output.json'), encoding='utf-8')
         out_msg = json.loads(out.read())
+        print('Comparing expected event to actual event:\n\n')
```

Comment:
Adding this to make troubleshooting these a bit less debug-required.        We need to use something other than nose1, and anything beyond this should go into that effort.